BasicSteps/TestPlanReference.cs

```diff
@@ -376,6 +376,122 @@ public void loadTestPlan()
             
             UpdateStep();
         }
+
+        string GetPath()
+        {
+            object testplandir = null;
+            var currentSerializer = TapSerializer.GetObjectDeserializer(this);
+            if (currentSerializer != null && currentSerializer.ReadPath != null)
+                testplandir = System.IO.Path.GetDirectoryName(currentSerializer.ReadPath);
+            
+            var refPlanPath = Filepath.Expand(testPlanDir: testplandir as string);
+            refPlanPath = refPlanPath.Replace('\\', '/');
+            if (!File.Exists(refPlanPath))
+            {
+                Log.Warning("File does not exist: \"{0}\"", refPlanPath);
+                return null;
+            }
+            return refPlanPath;
+        }
+
+        public bool anyStepsLoaded => ChildTestSteps.Any() && File.Exists(GetPath());
+
+        
+        
+        [Display("Are you sure?")]
+        class ConvertWarning
+        {
+            public enum ConvertOrCancel
+            {
+                Convert,
+                Cancel
+            }
+            [Browsable(true)]
+            [Layout(LayoutMode.FullRow)]
+            public string Message => "Are you sure you want to convert the Test Plan Reference to a Sequence?\n\nAny future changes to the referenced test plan will not be reflected.";
+
+            [Layout(LayoutMode.FloatBottom | LayoutMode.FullRow)]
+            [Submit]
+            public ConvertOrCancel Response { set; get; } = ConvertOrCancel.Cancel;
+
+        }
+            
+        
+        [Browsable(true)]
+        [Display("Convert to Sequence", "Convert  the test plan reference to a sequence step.", Order: 1.1)]
+        [EnabledIf(nameof(anyStepsLoaded), HideIfDisabled = true)]
+        public void ConvertToSequence()
+        {
+            var warn = new ConvertWarning();
+            UserInput.Request(warn, true);
+            if (warn.Response == ConvertWarning.ConvertOrCancel.Cancel)
+                return;
+            var subPlan = TestPlan.Load(GetPath());
```

Comment:
`GetPath()` returns null if the test plan can't be found. We should give a prettier error message than whatever `TestPlan.Load()` does with a null argument.

Reply:
`GetPath()` already logs a warning in the null case. It seems more correct to log the warning here